fractal/parser.py

```diff
@@ -470,7 +475,9 @@
 )
 workflow_edit_parser.add_argument("project_id", type=int, help="Project ID")
 workflow_edit_parser.add_argument("workflow_id", type=int, help="Workflow ID")
-workflow_edit_parser.add_argument("--name", type=str, help="New workflow name")
+workflow_edit_parser.add_argument(
+    "new_name", type=str, help="New workflow name"
```

Comment:
I would rather stick with the previous format (`--name`, to become `--new-name`), because this is how the other `edit` commands behave.

Reply:
Done.